Project.toml

```diff
@@ -3,8 +3,8 @@ uuid = "e88e6eb3-aa80-5325-afca-941959d7151f"
 version = "0.3.2"
 
 [deps]
+AbstractFFTs = "621f4979-c628-5d54-868e-fcf4e3e8185c"
 DiffRules = "b552c78f-8df3-52c6-915a-8e097449b14b"
-FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
```

Comment:
I think if you include FFTW under [extras] instead, then it should be available only when testing. If that doesn't work then try adding also a section like this at the end:  
```
[targets]
test = ["Test", "Distances", "StatsFuns", "FFTW"]
```